airflow/providers/amazon/provider.yaml

```diff
@@ -441,6 +447,8 @@ connection-types:
     connection-type: aws
   - hook-class-name: airflow.providers.amazon.aws.hooks.emr.EmrHook
     connection-type: emr
+  - hook-class-name: airflow.providers.amazon.aws.hooks.redshift.RedshiftSQLHook
+    connection-type: redshift+redshift_connector
```

Comment:
Is there a reason to not just have this as `redshift`? -- for example `AIRFLOW_CONN_MY_REDSHIFT=redshift://cluster/db` ?

Reply:
fixed in 1b22d59c1d3f25370b81c5f414a6e129a320e0ab